src/devices/schneider_electric.ts

```diff
@@ -311,6 +317,83 @@ const schneiderElectricExtend = {
             ],
         };
     },
+
+    dimmingMode: (): ModernExtend => {
+        const extend = enumLookup({
+            name: 'dimmer_mode',
+            lookup: {
+                Auto: 0,
+                'RL-LED': 3,
+            },
+            cluster: 'lightingBallastCfg',
+            attribute: 'wiserControlMode',
+            description: 'Controls the dimmer mode.',
+            entityCategory: 'config',
+        });
+        extend.configure.push(setupConfigureForReading('lightingBallastCfg', ['wiserControlMode']));
+        return extend;
+    },
+
+    occupancyConfiguration: (): ModernExtend => {
+        const extend = enumLookup({
+            name: 'occupancy_sensitivity',
+            lookup: {
+                Low: 50,
+                Medium: 75,
+                High: 100,
+            },
+            cluster: 'msOccupancySensing',
+            attribute: 'schneiderOccupancySensitivity',
+            description: 'Sensitivity of the occupancy sensor',
+            entityCategory: 'config',
+        });
+
+        const luxScale: ScaleFunction = (value: number, type: 'from' | 'to') => {
+            if (type === 'from') {
+                return Math.round(Math.pow(10, (value - 1) / 10000));
+            } else {
+                return Math.round(10000 * Math.log10(value) + 1);
+            }
+        };
+
+        const luxThresholdExtend = numeric({
+            name: 'ambience_light_threshold',
+            cluster: 'manuSpecificSchneiderOccupancyConfiguration',
+            attribute: 'ambienceLightThreshold',
+            reporting: {min: '10_SECONDS', max: '1_HOUR', change: 5},
+            description: 'Threshold above which occupancy will not trigger the light switch.',
```

Comment:
I'm not 100% sure about this definition. The manual claims it can be adjusted to avoid false detections when there are large changes in environmental lighting.

Reply:
I've double checked that this is how it works. Useful for not triggering the light during the day when the room is bright enough already.